pkg/ccl/backupccl/show.go

```diff
@@ -271,11 +285,16 @@ func showBackupPlanHook(
 	return fn, infoReader.header(), nil, false, nil
 }
 
+//backupShower specifies the table that SHOW BACKUP will print to the user's sql console.
+//header contains the schema of the table and fn organizes info from the unmarshalled
+//backup manifest into datums that the results channel can cleanly pipe, row wise,
+//to the console. The plan hook specifies the function, while showBackup calls the function.
 type backupShower struct {
 	header colinfo.ResultColumns
 	fn     func([]BackupManifest) ([]tree.Datums, error)
 }
 
+// backupShowerHeaders: defines the schema for the table presented to the user
```

Comment:
nit: remove colon after method name, and period at the end of comment.

Reply:
done.